src/tools/bundle-cli.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * @license
+ * Copyright (c) 2019 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import minimist from 'minimist';
```

Comment:
suggest replacing minimist with something else.  It's obsolete and unmaintained.

Reply:
sigh is already using minimist, and given the scope (basic args parsing) being unmaintained isn't much of a problem - we don't currently need sophisticated parsing and it's unlikely to break in the foreseeable future.

Reply:
Just to clarify: I'm not against replacing minimist with something newer, but it makes sense for this PR to use something we already depend on (and know about).

Reply:
I'll leave as is on the basis that that's what we already use in our repo. Let's migrate to a new solution across all usages.